packages/edit-site/src/components/page-pages/index.js

```diff
@@ -42,6 +48,8 @@ const defaultConfigPerViewType = {
 	},
 };
 
+const DefaultWidget = forwardRef( ( { render } ) => render );
```

Comment:
This is only necessary until other views (i.e. `GridView`) implement this `Widget` system.